asgs_main.sh

```diff
@@ -3381,9 +3383,7 @@ while [ true ]; do
                fi
                CURRENT_EVENT="FORE"
                CURRENT_STATE="CMPL"
-   	         RMQMessage "INFO" "$CURRENT_EVENT" "$THIS>$ENSTORM" "$CURRENT_STATE" "Forecast Complete for Adv=$ADVISORY Ens=$ENSTORM"
-               # send current run.properties to RMQ
-               #RMQMessageRunProp "$STORMDIR"
+   	           RMQMessage "INFO" "$CURRENT_EVENT" "$THIS>$ENSTORM" "$CURRENT_STATE" "Forecast Complete for Adv=$ADVISORY Ens=$ENSTORM"
```

Comment:
spacing change only, can be moved over and eliminated from this diff it looks like